packages/e2e/pages/base-page.ts

```diff
@@ -1,86 +1,86 @@
-import { type Locator, type Page, expect } from "@playwright/test";
+import { type Locator, type Page, expect } from '@playwright/test'
 
 export class BasePage {
-  readonly page: Page;
-  readonly connectWalletBtn: Locator;
-  readonly kepltWalletBtn: Locator;
-  readonly portfolioLink: Locator;
-  readonly poolsLink: Locator;
-  readonly walletBalance: Locator;
-  readonly connectedWalletBtn: Locator;
+  readonly page: Page
```

Comment:
is this biome vs prettier? Should we reformat the whole codebase in one go?

Reply:
This is prettier. I'm not sure why only the e2e consistently need prettier format. I'll investigate when possible